sd-boot/pkg.yaml

```diff
@@ -8,10 +8,13 @@ steps:
   - install:
     - |
       cd /tmp/systemd
-      mkdir -p /rootfs/
+      mkdir -p /rootfs/usr/lib/systemd/boot/efi
 
-      cp build/src/boot/*.efi /rootfs/
-      cp build/src/boot/*.efi.stub /rootfs/
+      cp build/src/boot/*.efi /rootfs/usr/lib/systemd/boot/efi
+      cp build/src/boot/*.efi.stub /rootfs/usr/lib/systemd/boot/efi
```

Comment:
Should we? This package is not installed at a rootfs, but used in Talos' Dockerfile to copy from, so it might be okay to put necessary stuff on the root, and do `COPY --from=pkg-sd-boot /*.efi /sd-boot-${TARGETARCH}.efi` instead of `COPY --from=pkg-sd-boot /usr/lib/systemd/boot/efi/*.efi /sd-boot-${TARGETARCH}.efi`

It's okay to make it exempt from fhs if it's not used like that

Reply:
yeh, makes sense, keep as is and drop the test